hs_core/hydroshare/users.py

```diff
@@ -310,6 +179,7 @@ def list_users(query=None, status=None, start=None, count=None):
     return qs
 
 
+# Pabitra: This one seems to be not used anywhere. Can I delete it?
```

Comment:
The more I look at this function, I find it not useful at all. I am going to delete it. If we ever need something like this it won't be hard to add this functionality.